.github/workflows/historic-data-migrator.yml

```
@@ -0,0 +1,42 @@
+---
+name: "Historic data migrator"
```

```suggestion
name: Historic Data Migration
```

Typo and quotes aren't needed

---

Historic Data Migrator*